polaris-react/src/components/Modal/components/CloseButton/CloseButton.scss

```diff
@@ -16,44 +16,19 @@
     svg {
       fill: var(--p-color-icon-hover);
     }
-
-    #{$se23} & {
-      background-color: var(--p-color-bg-transparent-hover-experimental);
-    }
```

Comment:
The close button renders a `<Button>` in se23 mode, so this whole css file is unused. Therefore, we can remove these style overrides 🎉